gui/CMakeLists.txt

```diff
@@ -38,6 +38,7 @@ vole_compile_library(
 	dist_view/distviewmodel
 	dist_view/viewer_tasks
 	dist_view/foldingbar
+	dist_view/actionbutton
```

Comment:
Move actionbutton to widgets/. it is general enough to be used also in other parts of the GUI and probably will be soon in bandview.